### PR TITLE
docs: add bus-odd arch discussion summary

### PR DESCRIPTION
**Pull request by TakaHoribe**
add bus odd arch discussion summary for planning/control/vehicle as of today.

This will be updated through discussion.

### CODE REVIEW

File: docs/design/component-interfaces/planning-control-vehicle-interface.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+      - [geometry_msgs::msg::AccelWithCovariance](https://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/AccelWithCovariance.html) initial_acceleration
+      - sequence<[autoware_auto_perception_msgs::msg::PredictedPath](https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/blob/master/autoware_auto_perception_msgs/msg/PredictedPath.idl), 10> predicted_paths
+        - sequence<[geometry_msgs::msg::Pose](https://docs.ros.org/en/lunar/api/geometry_msgs/html/msg/Pose.html), 100> path
+        - builtin_interfaces::msg::Duration time_step
+        - float confidence
+    - sequence<[autoware_auto_perception_msgs::msg::Shape](https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/blob/master/autoware_auto_perception_msgs/msg/Shape.idl), 5> shape
+      - [geometry_msgs::msg::Polygon](https://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Polygon.html) polygon
+      - float height
+
+#### (NEW) Traffic Light Response
```
Comment (mitsudome-r) [inline docs/design/component-interfaces/planning-control-vehicle-interface.md:33]: I don't think we need to mention (NEW) in the design document.
Same with the other sections.

**Review by mitsudome-r — APPROVED** [review]

File: docs/design/component-interfaces/planning-control-vehicle-interface.md (hunk 1 above)
Comment (TakaHoribe) [reply docs/design/component-interfaces/planning-control-vehicle-interface.md]: Thanks. Addressed in https://github.com/autowarefoundation/autoware-documentation/pull/48/commits/90ce7eaa8ec2a1fa1c57c6d3c9b37435ca769a8a

**Review by mitsudome-r — DISMISSED** [review]
LGTM, I will create another PR to add interface for other components (e.g. localization, sensing, perception)

**Review by mitsudome-r — APPROVED** [review]

**Comment by mitsudome-r** [comment]
The CI failure is from the document that was not modified in the PR. I will merge this and the CI should be fixed with another PR.

**Pull request merged**